pkg/object/graph/depends_test.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/stretchr/testify/assert"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"sigs.k8s.io/cli-utils/pkg/object"
+	depsutil "sigs.k8s.io/cli-utils/pkg/object/dependson/testutil"
```

Comment:
This refactor helps mitigate an import cycle and mirrors the pattern used for the dependson and applyt-time-mutation mutators (with an unfortunate naming overlap).